typescript/core/src/v0/ChunkCursor.ts

```diff
@@ -64,65 +64,18 @@ export class ChunkCursor {
       throw new Error("Cannot compare a reversed ChunkCursor to a non-reversed ChunkCursor");
     }
 
-    if (reverse) {
-      // If chunks don't overlap at all, sort later chunk first
-      if (this.chunkIndex.messageEndTime < other.chunkIndex.messageStartTime) {
-        return 1;
-      }
-      if (this.chunkIndex.messageStartTime > other.chunkIndex.messageEndTime) {
-        return -1;
-      }
-    } else {
-      // If chunks don't overlap at all, sort earlier chunk first
-      if (this.chunkIndex.messageEndTime < other.chunkIndex.messageStartTime) {
-        return -1;
-      }
-      if (this.chunkIndex.messageStartTime > other.chunkIndex.messageEndTime) {
-        return 1;
+    const diff = Number(this.getSortTime() - other.getSortTime());
+    if (diff === 0) {
+      if (reverse) {
+        // Break ties by chunk offset in the file
+        return Number(other.chunkIndex.chunkStartOffset - this.chunkIndex.chunkStartOffset);
```

Comment:
I'm not sure why this isn't simpler:
```javascript
if (diff === 0) {
    diff = Number(this.chunkIndex.chunkStartOffset - other.chunkIndex.chunkStartOffset);
}
```

Reply:
I guess you'd have to make diff not-const.

Reply:
simplified per recommendation